src/main/java/org/jitsi/jicofo/JitsiMeetConferenceImpl.java

```diff
@@ -1407,7 +1407,7 @@ public XMPPError onSessionAccept(
         {
             String errorMsg
                 = "No participant found for: " + participantJid;
-            logger.error(errorMsg);
+            logger.warn(errorMsg);
```

Comment:
this seems pretty bad to me, shouldn't it be error?  what's the situation it's happening where it isn't bad?

Reply:
When looking at the code it can happen after participant was marked as expired by Jicofo(removed from participants list), because it didn't reply with an ack to the session-initiate. So lets say participant lost connectivity for a while and then came back, but as the current code goes this one is not recoverable - jicofo will not re-invite this participant(maybe should).

Reply:
i just worry this is going to make real issues harder to find.

Reply:
Yeah, the point is that currently those messages appear in the logs and nobody does anything about them. So do you think anyone would do in future? How would you know it's a bigger issue this time? It will still appear on the warning level.

Reply:
I wanted to have the error level reserved for cases which when happen really mean something's wrong and there's no ambiguity. This case can happen during "normal" operation lets say.